tensorrt_llm/llmapi/llm_args.py

```diff
@@ -680,7 +683,10 @@ def model_name(self) -> Union[str, Path]:
         return self.model if isinstance(self.model, str) else None
 
 
-class LlmArgs(BaseModel):
+class LlmArgsBase(BaseModel):
```

Comment:
How about renaming to `BaseLlmArgs`? The `Base` prefix is more clear to signal the class's role as a Base/Parent class.

Reply:
Good suggestion, done, thanks.